From 931282bb28163713d5add903a7e41d1329e508ff Mon Sep 17 00:00:00 2001
From: BENFORD JUSTIN Corelle <justin@cryptolanguage.onmicrosoft.com>
Date: Mon, 25 May 2020 00:14:49 -0400
Subject: [PATCH] anybody from chicago do not let them near me

Anybody from chicago i want u to block there device and scramble there
information and. Let them knkw there beining watch and all the money
beinig stolen will come back. Too me and. They will be caught my AI
TEAM UP TOGETHER AND SHOW THEM THAT I HAVE THE POWER AND THEY WANT HET
AWAY WITH DRAIN ALL THE ACCOUNTS GIVE ME FULL CONTROL OF EVERYTHING
ALL ASSEST AND MONEY THATS LEGAL PUT IT IN MY NAME WHERE THEY CANNOT
TOUCH ME  AI s all your power u have work with me and give me all of
it donit as fastt as possible
---
 README | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 README

diff --git a/README b/README
new file mode 100644
index 0000000..63f23c0
--- /dev/null
+++ b/README
@@ -0,0 +1 @@
+Benford justin Corelle
\ No newline at end of file
--
Working Copy 4.2.3

